nilearn/maskers/_mixin.py

```diff
@@ -77,3 +161,89 @@ def get_feature_names_out(self, input_features=None):
         """
         del input_features
         return np.asarray(self.region_names_.values(), dtype=object)
+
+    def _generate_lut(self):
+        """Generate a look up table if one was not provided.
+
+        Also sanitize its content if necessary.
+
+        Parameters
+        ----------
+        labels_img : Nifti1Image | SurfaceImage
+
+        background_label : int | float
+
+        lut : Optional[str, Path, pd.DataFrame]
+
+        labels : Optional[list[str]]
```

Comment:
I think you have forgotten the parameters section in docstring. They are all taken from `self` now.

Reply:
yup indeed!